src/wp-admin/includes/export.php

```diff
@@ -243,6 +243,8 @@ function export_wp( $args = array() ) {
 	 * @return string
 	 */
 	function wxr_cdata( $str ) {
+		$str = (string) $str;
```

Comment:
This would be safer:

```suggestion
		if ( ! is_string( $str ) ) {
			$str = '';
		}
```

Reply:
Of even, why not just:


```suggestion
		if ( ! is_string( $str ) ) {
			return '';
		}
```

If the value is an empty string, there's nothing needing to be wrapped in a CDATA section.

Reply:
Done.